content/js/users.md

```diff
@@ -61,6 +73,19 @@ skygear.changePassword(oldPassword, newPassword).then((user) => {
 });
 ```
 
+## Change email
+
+``` javascript
+skygear.saveUser({
+  id: '<your-user-id>', // such as skygear.currentUser.id
+  email: '<your-new-email-address>'
+}).then((user) => {
+  console.log('Email updated to:', user.email);
+}, (error) => {
+  console.error(error);
+});
+```
```

Comment:
@rickmak I hope I'm understanding it right: there are two tables to handle user information; one table keeps track of userid, username, email and password for login purposes; the other table keeps track of user profile and can be accessed using the `skygear.publicDB.query` syntax. However, when I actually tried the `skygear.saveUser` function, I get:
``` javascript
{
  "status": 400,
  "error": {
    "name": "InvalidArgument",
    "code": 108,
    "message": "Cannot assign user role on AcceesModel is not RoleBaseAccess",
    "info": { "arguments": [ "roles" ] }
  }
}
```
Should I open up a new issue under skygear-JS-SDK repo for this?